<ui version="4.0" >
 <class>MBSWcontent_Form</class>
 <widget class="QWidget" name="MBSWcontent_Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>475</width>
    <height>200</height>
   </size>
  </property>
  <property name="font" >
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle" >
   <string>Measuring Blocks</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <property name="spacing" >
      <number>-1</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="MBSWtitle_label" >
       <property name="minimumSize" >
        <size>
         <width>200</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>16</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text" >
        <string>Selected Measuring Blocks:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="MBSWrefreshTimeTitle_label" >
       <property name="minimumSize" >
        <size>
         <width>140</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>16</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <family>Liberation Sans</family>
        </font>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="MBSWrefreshTimeValue_label" >
       <property name="minimumSize" >
        <size>
         <width>55</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>16</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <family>Liberation Sans</family>
        </font>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="timemode_pushButton" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../resources/FreeSSM.qrc" >
         <normaloff>:/icons/oxygen/16x16/chronometer.png</normaloff>:/icons/oxygen/16x16/chronometer.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="selectedMBsSWs_tableWidget" >
     <property name="minimumSize" >
      <size>
       <width>382</width>
       <height>123</height>
      </size>
     </property>
     <property name="font" >
      <font>
       <family>Liberation Sans</family>
      </font>
     </property>
     <property name="editTriggers" >
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::ContiguousSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text" >
       <string>Title:</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Value:</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Unit:</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <property name="spacing" >
      <number>10</number>
     </property>
     <property name="topMargin" >
      <number>8</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="startstopmbreading_pushButton" >
       <property name="minimumSize" >
        <size>
         <width>70</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>70</width>
         <height>32</height>
        </size>
       </property>
       <property name="text" >
        <string> Start  </string>
       </property>
       <property name="icon" >
        <iconset resource="../resources/FreeSSM.qrc" >
         <normaloff>:/icons/chrystal/32x32/player_play.png</normaloff>:/icons/chrystal/32x32/player_play.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mbswdelete_pushButton" >
       <property name="minimumSize" >
        <size>
         <width>40</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>32</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../resources/FreeSSM.qrc" >
         <normaloff>:/icons/chrystal/32x32/edit_remove.png</normaloff>:/icons/chrystal/32x32/edit_remove.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mbswadd_pushButton" >
       <property name="minimumSize" >
        <size>
         <width>40</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>32</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../resources/FreeSSM.qrc" >
         <normaloff>:/icons/chrystal/32x32/add.png</normaloff>:/icons/chrystal/32x32/add.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mbswmoveup_pushButton" >
       <property name="minimumSize" >
        <size>
         <width>40</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>32</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../resources/FreeSSM.qrc" >
         <normaloff>:/icons/chrystal/32x32/1uparrow.png</normaloff>:/icons/chrystal/32x32/1uparrow.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mbswmovedown_pushButton" >
       <property name="minimumSize" >
        <size>
         <width>40</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>32</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../resources/FreeSSM.qrc" >
         <normaloff>:/icons/chrystal/32x32/1downarrow.png</normaloff>:/icons/chrystal/32x32/1downarrow.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/FreeSSM.qrc" />
 </resources>
 <connections/>
</ui>
